src/StateLabel.tsx

```diff
@@ -37,7 +77,7 @@ const StateLabelBase = styled.span<StyledStateLabelBaseProps>`
   align-items: center;
   font-weight: 600;
   line-height: 16px;
-  color: ${get('colors.white')};
+  color: ${get('colors.bg.primary')};
   text-align: center;
   border-radius: ${get('radii.3')};
```

Comment:
Since we're only setting `borderColor` in the color variants, we'll need to set the border width and style here:
```suggestion
  border-radius: ${get('radii.3')};
  border-width: 1px;
  border-style: solid;
```